package.json

```diff
@@ -54,19 +58,23 @@
   "optionalDependencies": {
     "chokidar": "^1.6.0"
   },
+  "_moduleAliases": {
+    "babel-register": "@babel/register"
```

Comment:
I wonder is it used anywhere? I see all instances pointing to the `@babel/register`...

Reply:
I'll be able to remove it when istanbuljs fully supports babel 7.

Reply:
Oh, okey.